iroh-sync/src/net.rs

```diff
@@ -103,11 +152,55 @@ where
         .read_to_end(0)
         .await
         .map_err(|error| AcceptError::close(peer, namespace, error))?;
+
+    let t_process = t_start.elapsed() - t_connect;
     let namespace = res?;
 
-    debug!(?peer, ?namespace, "sync[accept]: done");
+    debug!(
+        peer = %peer.fmt_short(),
+        namespace = %namespace.fmt_short(),
+        ?t_process,
+        ?t_connect,
+        sent = %progress.num_sent,
+        recv = %progress.num_recv,
+        "sync[accept]: done, success"
+    );
+
+    let timings = Timings {
```

Comment:
We can remove this again also. Added for debugging purposes. But might be useful nevertheless..

Reply:
I like it, alternatively we could make these into metrics